include/crocoddyl/core/state-base.hpp

```diff
@@ -50,13 +51,15 @@ class StateAbstractTpl {
                      Jcomponent firstsecond = both) const = 0;
   virtual void Jintegrate(const Eigen::Ref<const VectorXs>& x, const Eigen::Ref<const VectorXs>& dx,
                           Eigen::Ref<MatrixXs> Jfirst, Eigen::Ref<MatrixXs> Jsecond,
-                          Jcomponent firstsecond = both) const = 0;
+                          const Jcomponent firstsecond = both, const AssignmentOp = setto) const = 0;
```

Comment:
@proyan we don't have any documentation inside Crocoddyl, but I want to make this team effort.

I am not asking you to document this class, but could you add documentation for these changes (i.e. `Jintegrate` and `JintegrateTransport`)?

Reply:
For now, just do it only for the base class.

Reply:
:+1: